scripts/allowlist_utils.js

```diff
@@ -36,33 +36,33 @@ function isInitialAdminAccountsAvailable() {
     return process.env.INITIAL_ADMIN_ACCOUNTS;
 }
 
-function isInitialWhitelistedAccountsAvailable() {
-    return process.env.INITIAL_WHITELISTED_ACCOUNTS;
```

Comment:
If we stop supporting the old variable, we'll be making this backwards-incompatible what will require a version bump. It might be worth checking with Madeline if she is ok with that.

Reply:
added support for both environment variables